### DISCUSSION TITLE
RealESRGAN_x4Plus Anime 6B for x2

### DISCUSSION THREAD

**Discussion by Kuphi** [xinntao/Real-ESRGAN#840] [Q&A]
I really love the results of **RealESRGAN_x4Plus Anime 6B** for anime!

But now I have 1080p anime material and only need x2. I'm currently using **RealESRGANv2 AnimeVideo xs-x2**, but the results are not as good as the results of RealESRGAN_x4Plus Anime 6B.

To clarify: Upscaling 480p to 4k using **RealESRGAN_x4Plus Anime 6B** has better peak results than upscaling from 1080p to 4k using **RealESRGANv2 AnimeVideo xs-x2**. Due to the lower quality of a 480p video, upscaling from 1080p ofc has better results at the lower end.

Anyone knows of a **RealESRGAN_x2Plus Anime 6B** model for x2? Pretty sure the results would be awesome. Thanks!

**Comment by Kuphi** [comment]
1080p 2x with **RealESRGANv2 AnimeVideo xs-x2**:
![vlcsnap-2024-08-27-20h59m45s473](https://github.com/user-attachments/assets/6691377a-7546-4834-a37f-7f42d639ec69)
bold outlines

1080p downscaled to 540p and then 4x with **RealESRGAN_x4Plus Anime 6B**
![vlcsnap-2024-08-27-20h59m42s743](https://github.com/user-attachments/assets/b8f12b78-29ad-4ddc-8db9-b48c64a343cf)
Despite downscaling to 540p before upscaling, the result is visually more appealing, though some detail was lost.

**Comment by unamed666** [comment]
How about RealESRGAN AnimeVideo-v3 ?
It has x2 x3 and x4
